src/modules/thumbnail/thumbnail.model.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  Column,
+  Model,
+  Table,
+  PrimaryKey,
+  ForeignKey,
+  DataType,
+  BelongsTo,
+} from 'sequelize-typescript';
+
+import { ThumbnailAttributes } from './thumbnail.attributes';
+import { FileModel } from '../file/file.model';
+
+@Table({
+  underscored: true,
+  timestamps: true,
+  tableName: 'thumbnails',
+})
+export class ThumbnailModel extends Model implements ThumbnailAttributes {
+  @PrimaryKey
+  @Column
+  id: number;
+
+  @ForeignKey(() => FileModel)
+  @Column(DataType.INTEGER)
+  fileId: number;
+
+  @BelongsTo(() => FileModel, 'fileId')
```

Comment:
We relate `thumbnails.fileId` with `files.id`
`@BelongsTo(() => FileModel, 'id')`